BackendServices/app/controllers/CompanyController.java

```diff
@@ -1,38 +1,30 @@
 package controllers;
 
-import play.*;
-import play.libs.Json;
-import play.mvc.*;
-import views.html.*;
+import play.mvc.Controller;
 
 import com.fasterxml.jackson.databind.JsonNode;
-import com.fasterxml.jackson.databind.node.ObjectNode;
 
-public class CompanyController extends Controller {
 
-	public static Result index() {
-		return ok(index.render("Your new application is ready."));
-	}
+public class CompanyController extends Controller {
 
-	public static Result sayHello() {
+	@BodyParser.Of(BodyParser.Json.class)
+	public static Result createcontrolType() {		
 		JsonNode json = request().body().asJson();
-		if(json == null) {
-			return badRequest("Expecting Json data");
+		System.out.println(json);
+		if (json==null) {
+			return badRequest("Missing json ");
+		}
+		
+		String type = json.findPath("name").textValue();
+		if(type == null) {
+			return badRequest("Missing parameter [name]");
 		} else {
-			String name = json.findPath("name").textValue();
-			if(name == null) {
-				return badRequest("Missing parameter [name]");
-			} else {
-				return ok("Hello " + name);
-			}
+			Company service = Company.create(name);
```

Comment:
rename this variable to something else, Company service ??, you should use the example code as a reference not copy and  edit.